### PR TITLE
feat : login & signup

### PR DESCRIPTION
**Pull request by sudipkundu999**

### CODE REVIEW

**Comment by itsmebipul00** [comment]
pngs are the heaviest of all image files, so it would be better if you use others

**Review by itsmebipul00 — APPROVED** [review]

File: pages/login/login.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+header {
+  background-color: #ffffff;
+  height: 8vh;
+}
+
+body {
+  background-color: #f7f7f7;
+}
```
Comment (rutu14) [inline pages/login/login.css:8]: Assign Variables  for color codes.

File: pages/login/login.html [hunk 2]
```diff
@@ -1,12 +1,99 @@
...
+            <h3>CobraStore</h3>
+          </div>
+        </div>
+        <div class="cui-nav-search">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
+            <path
+              d="M500.3 443.7l-119.7-119.7c27.22-40.41 40.65-90.9 33.46-144.7C401.8 87.79 326.8 13.32 235.2 1.723C99.01-15.51-15.51 99.01 1.724 235.2c11.6 91.64 86.08 166.7 177.6 178.9c53.8 7.189 104.3-6.236 144.7-33.46l119.7 119.7c15.62 15.62 40.95 15.62 56.57 0C515.9 484.7 515.9 459.3 500.3 443.7zM79.1 208c0-70.58 57.42-128 128-128s128 57.42 128 128c0 70.58-57.42 128-128 128S79.1 278.6 79.1 208z"
+            />
+          </svg>
+          <input type="text" placeholder="Search" />
```
Comment (rutu14) [inline pages/login/login.html:45]: This can have a button to submit the search query.

File: pages/login/login.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+header {
+  background-color: #ffffff;
```
Comment (zoef-yt) [inline pages/login/login.css:2]: Use var for colours.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: pages/login/login.css (hunk 3 above)
Comment (sudipkundu999) [reply pages/login/login.css]: Done

File: pages/login/login.css (hunk 1 above)
Comment (sudipkundu999) [reply pages/login/login.css:13]: Done

File: pages/login/login.html (hunk 2 above)
Comment (sudipkundu999) [reply pages/login/login.html:45]: Yes! The whole navbar will change.
I have taken inspiration from your page.
I will fix it during refactoring.

Comment (rutu14) [reply pages/login/login.html:45]: Cool

**Pull request merged**